demisto_sdk/commands/init/tests/initiator_test.py

```diff
@@ -614,5 +608,3 @@ def test_script_init_with_ignore_secrets(mocker, monkeypatch, initiator, tmpdir)
     assert {f"{SCRIPT_NAME}.py", f"{SCRIPT_NAME}.yml", f"{SCRIPT_NAME}_test.py",
             "README.md", "test_data"} == script_dir_files
     assert os.stat(secrets_ignore_path).st_size > 0
-    shutil.rmtree('Packs')
```

Comment:
Why did you remove these lines?

Reply:
because I changed that I create the files in a temporary  repo (with ChangeCWD(tmpdir)) that removes the files automatically.